builds.yml

```diff
@@ -74,7 +74,7 @@ buildTypes:
     features:
       - build-flask
       - keyring-snaps
-      # - bitcoin
+      - bitcoin
```

Comment:
This should not be part of this PR IMO

Reply:
oops thanks for reviewing again! I checked out the builds.yml and brought some unwanted changes, I'll verify everything.